packages/forma-36-website/src/pages/foundation/typography/design.mdx

```diff
@@ -0,0 +1,128 @@
+
+These are the font family we use in our systems...
+
+![Typefaces demonstration, San Francisco, Lucida Grande, Neue Helvetica, Arial, and Segoe UI](./assets/typeface.png)
+
+Our font family consists of different font types, they are known as *system fonts*, 
+every operating system comes with at least one default font...
+
+**San Francisco** appears on Safari (Mac OS X and iOS); **Neue Helvetica** and **Lucida Grande** appears on older versions of Mac OS X; 
+**Segoe UI** targets Windows and Windows phone; **Arial** is available on almost all operating systems.
+
+<br />
+
+## Font sizes
+Pixels units maintainence is rigid. Hence, we use *REM* units to define our font sizes.
+
+### What are REMs?
+REMs (root em) are simply a ratio of the root element font size. The root element here is the browser default font size 16px. (1rem = 16px) (2rem = 32px)
+
+Using rem units enables our system to scale up/down based on user's prefrence.
+
+
+<br />
+
+### Available Sizes
+
+<br />
+<SectionHeading>Section Heading</SectionHeading>
+
+*Section Heading* addresses the small blocks that requires extra context, to make it easier for audience to understand.
+
+Font size: **12px** - Line Height: **18px** - Weight: **600**
+
+<br />
+
+<Paragraph>Body</Paragraph>
+
+*Body* is the base font size for content. Used usually for paragraphs.
+
+Font size: **14px** - Line Height: **21px** - Weight: **500**
+
+<br />
+
+<Subheading>Subheading</Subheading>
+
+*Subheadings* are headings for nested blocks within your sections; Assuming you have a section with heading "Contact us", you want to group "Address" related elements in a block, that when *Subheading* comes in handy.
+
+ Font size: **16px** - Line Height: **24px** - Weight: **700**
+
+<br />
+
+<Heading>Heading</Heading>
+
+*Heading* is used to define sections in your solution, this is **cruical** for accessibility; Headings are used to define sections/blocks in your page, it guides users through your page/view.
+
+Similarly visually impaired people use screen readers, and screen readers find sections in your page/view based on your heading hierarchy.
+
+Font size: **21px** - Line Height: **31px** - Weight: **700**
+
+<br />
+
+<DisplayText>Display</DisplayText>
+
+Use *Display* on similar cases as Display Text Large.
+
+Font size: **28px** - Line Height: **35px** - Weight: **700**
+
+<br />
+
+<DisplayText size='large'>Display Large</DisplayText>
+
+*Display Large* is used to display text in special scenarios - use Display Text Large on empty states, promotional/featured items, etc.
+
+*Display Large* should not be used as a replacement for headings/page titles (use the Heading styles instead).
+
+Font size: **35px** - Line Height: **44px** - Weight: **700**
+
+<br />
+
+### Line height
+
+Line height are basically verticle wrappers around letters on each line. 
+
+Our line height is also defined based on **rem**, this forces the line height to correspond to the root font size as well.
```

Comment:
Line height can be described as a vertical padding around letters on each line. 

The Forma 36 line height is defined by **rem**, which forces the line height to correspond to the root font size.